bundles/ch.elexis.pdfBills/src/ch/elexis/pdfBills/ElexisPDFGenerator.java

```diff
@@ -258,6 +261,87 @@ private void generatePdf(File styleSheet, File output, boolean withQr) {
 		}
 	}
 
+	private String getMessagePDFText(final InvoiceState invoiceState) {
+		String billMsg = "";
+		String invStateTxt = rechnung.getInvoiceState().toString();
+		ch.elexis.data.Fall.Tiers tiers = rechnung.getFall().getTiersType();
+		
+		switch (invoiceState) {
+		case UNKNOWN:
+		case IN_EXECUTION:
+		case STOP_LEGAL_PROCEEDING:
+		case OWING:
+		case PARTIAL_LOSS:
+		case TOTAL_LOSS:
+		case NOT_BILLED:
+		case ONGOING:
+		case TO_PRINT:
+		case NOT_FROM_YOU:
+		case NOT_FROM_TODAY:
+		case FROM_TODAY:
+		case EXCESSIVE_PAYMENT:
+		case REJECTED:
+		case BILLED:
+		case PARTIAL_PAYMENT:
+		case PAID:
+		case OPEN_AND_PRINTED:
+		case OPEN:
+			if (tiers == Fall.Tiers.GARANT) {
```

Comment:
Das könntest du hier sehr stark kürzen, dann wärs viel besser lesbar, so in der Art

```
String key = (Fall.Tiers.Garant == tiers) ? RnOutputter.CFG_MSGTEXT_TG_M0 : RnOutputter.CFG_MSGTEXT_TP_M0;
return CoreHub.globalCfg.get(key, invStateTxt);
```